app.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
@rwajon just want to confirm if `develop` branch is deployed automatically.
What about `staging` and how does it work?

Reply:
I've configured the Heroku app to automatically put a raised PR in the staging mode

Reply:
Any branch?